generated/PYPI_README.md

```diff
@@ -56,7 +56,7 @@ Apache Airflow is tested with:
 
 |            | Main version (dev)         | Stable version (2.10.2)    |
 |------------|----------------------------|----------------------------|
-| Python     | 3.8, 3.9, 3.10, 3.11, 3.12 | 3.8, 3.9, 3.10, 3.11, 3.12 |
+| Python     | 3.9, 3.10, 3.11, 3.12      | 3.9, 3.10, 3.11, 3.12      |
```

Comment:
Same here 2.10.2 still aupports Python 3.8 and 2.10.3 still should IMHO.

Reply:
Aaah, then I mis-interpreted the chapter in docs/apache-airflow/installation/supported-versions.rst:55,,, Minor/Major drop the Py versions but patch release keep them... will 2.11 be an exception as we mentally keep the 2.10 line or would be drop it in 2.11 as well?